src/screens/AddSiteScreen.js

```diff
@@ -67,84 +78,85 @@ class AddSiteScreen extends Component {
                                   <FormLabel>Title</FormLabel>
 
                                     <FormInput
-                                      placeholder="Add site title" />
-                                  <FormLabel>Description</FormLabel>
+                                      placeholder={add_site_title} />
+                                    <FormLabel>{description}</FormLabel>
                                     <FormInput
-                                      placeholder="Enter a description of the site"
+                                      placeholder={description_placeholder}
                                       containerStyle={largeTextInput}
                                       multiline={true}
                                       maxLength={40}
                                       maxHeight={50}
                                       editable={true}
                                        />
-                                  <FormLabel>Directions</FormLabel>
+                                     <FormLabel>{directions_placeholder}</FormLabel>
                                     <FormInput
-                                      placeholder="Enter a descriptive directions"
+                                      placeholder={directions_placeholder}
                                       containerStyle={largeTextInput}
                                       multiline={true}
                                       maxLength={40}
                                       maxHeight={50}
                                       editable={true}
                                        />
 
-                                  <FormLabel>Nearest Town</FormLabel>
+                                     <FormLabel>{nearest_town}</FormLabel>
                                     <FormInput
-                                      placeholder="Enter nearest/closest town"
+                                      placeholder={nearest_town_placeholder}
                                       editable={true}
                                       />
-                                  <FormLabel>Accessiblitiy</FormLabel>
+                                    <FormLabel>{accessibility}</FormLabel>
                                           <Picker>
-                                            <Picker.Item label="Paved Road" value="paved_road" />
-                                            <Picker.Item label="Dirt Road " value="dirt_road" />
-                                            <Picker.Item label="Uneven Terrain" value="uneven_terrain" />
-                                            <Picker.Item label="4x4(recommended)" value="4x4" />
-                                            <Picker.Item label="4X4 Drive High Clearance" value="high_clearence" />
-                                            <Picker.Item label="Hike" value="hike" />
+                                            <Picker.Item label={paved_road} value="paved_road" />
```

Comment:
Now, I am thinking I shouldn't of left the hardcoded values?

Reply:
You could either have them listed like this, or you could do a `_.map()` over an array of text values. For a short list like this, we're probably fine.

Reply:
Actually, now that I look again... doing `value="paved_road"` will probably just print `"paved_road"` won't it? Doesn't that need to be `value={paved_road}` ?